sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/ReceiverTests.cs

```diff
@@ -348,5 +351,31 @@ public async Task CreatingReceiverWithIdentifierSetsIdentifier()
             var identifier = receiver.Identifier;
             Assert.AreEqual(setIdentifier, identifier);
         }
+
+        [Test]
+        public async Task DeadLetterMessageChecksArguments()
```

Comment:
We should also add a test case where the values are specified in properties and parameter, but they are the same so we are not expecting an exception.